docs/sources/developers/http_api/serviceaccount.md

```diff
@@ -138,9 +138,9 @@ Content-Type: application/json
 
 See note in the [introduction]({{< ref "#service-account-api" >}}) for an explanation.
 
-| Action               | Scope                |
-| -------------------- | -------------------- |
-| serviceaccounts:read | serviceaccounts:id:1 |
+| Action               | Scope                 |
+| -------------------- | --------------------- |
+| serviceaccounts:read | serviceaccounts:id:\* |
```

Comment:
prettier is adding the `\` - how can I stop it doing this? As I think it's very confusing, the correct is just to have `*`